test/test_feed.cpp

```diff
@@ -19,30 +18,27 @@ using std::string;
 etree::Element atomFeed("x");
 etree::Element rss20Feed("x");
 
-MU_SETUP(loadFeeds)
+TEST_CASE("loadFeeds", "[feed]")
```

Comment:
It does behave as before, albeit with the slightly strange  effect that all the "[feed]" tests are now seen as a single test case.

The alternative is to use a more traditional test fixture, which Catch can also support:

https://github.com/philsquared/Catch/blob/master/docs/test-fixtures.md